app/src/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -2554,6 +2592,18 @@ private int getFontSizeInPx() {
                 getResources().getDisplayMetrics());
     }
 
+    private void reportFormEntry() {
+        TimedStatsTracker.registerEnterForm(getCurrentFormTitle());
+    }
+
+    private void reportFormExit() {
+        TimedStatsTracker.registerExitForm(getCurrentFormTitle());
+    }
```

Comment:
Do we care if the form was completed or not? I think `reportFormExit` is called even if the form wasn't completed or even saved.